pkg/splunk/enterprise/util.go

```diff
@@ -1707,26 +1708,23 @@ func validateMonitoringConsoleRef(c splcommon.ControllerClient, revised *appsv1.
 }
 
 // setInstallStateForClusterScopedApps sets the install state for cluster scoped apps
-func setInstallStateForClusterScopedApps(appDeployContext *enterpriseApi.AppDeploymentContext, status enterpriseApi.AppPhaseStatusType) {
+func setInstallStateForClusterScopedApps(appDeployContext *enterpriseApi.AppDeploymentContext) {
```

Comment:
nit: I think it would make more sense to pass install state as argument to this function so that it becomes a generic function to set any install state

Reply:
As the function name is already resembling the install part, and also, we don't have any use case for bulk transition., it's redundant to pass. Also, we don't have any use case of bulk transition, (particularly based on the scope) for other phases for now. So, we should be good.